### DISCUSSION TITLE
A little tool to check if you're snapchat detected

### DISCUSSION THREAD

**Discussion by ghost** [rodit/SnapMod#292] [General]
https://github.com/JReverse/Am-I-SC-Detected

dumped from latest natives binary.

Only Layer 1 Checks.

**Comment by RevealedSoulEven** [comment]
This is insane

**Comment by NMBlanco** [comment]
I don't get whats the sense of this?

**Reply by authorisation** [reply]
afaik Snapchat checks for specific paths and if they exist it won't let you login. This tool just checks if these paths exist and will tell you if you would in theory pass Layer 1.

**Comment by RevealedSoulEven** [comment]
@JReverse #301 any idea?

**Comment by RevealedSoulEven** [comment]
I think they filter it on the basis of tiers. Like if it's too confidential
then it won't let you login, or if either of them are fair, it will login.

As all devices won't be compatible and snapchat don't want to make the
situation worse.

**Reply by authorisation** [reply]
Layer 1 is just checking if you have Xposed or anything similar, that doesn't have to do anything with devices and their compatibility. I'm not sure but I think they also check SafetyNet there.

**Reply by Lucky-069** [reply]
Hey @JReverse @authorisation since the project has almost been abandoned and I'm now so used to this project that I just hate switching to stock snapchat. There's another snapchat mod available by fouadmods but it modifies the snapchat app itself unlike using xposed module to hook previously when Casper existed it got noticed by snapchat team and I got temporary ban twice and had to delete the app.. I wonder if they might still do it. Xposed module is different in a way tho, unhooking the app switches it back to stock..
Also if either of you could update the app that would be great!
